packages/@sanity/portable-text-editor/src/editor/plugins/__tests__/withPortableTextMarkModel.test.tsx

```diff
@@ -360,23 +360,25 @@ describe('plugin:withPortableTextMarksModel', () => {
           })
           PortableTextEditor.toggleMark(editor, 'bold')
           expect(PortableTextEditor.getValue(editor)).toMatchInlineSnapshot(`
-        Array [
-          Object {
-            "_key": "a",
-            "_type": "myTestBlockType",
-            "children": Array [
-              Object {
-                "_key": "a1",
-                "_type": "span",
-                "marks": Array [],
-                "text": "1234",
-              },
-            ],
-            "markDefs": Array [],
-            "style": "normal",
-          },
-        ]
-      `)
+Array [
```

Comment:
The behaviour has changed here.
Previously if I highlighted '**1**23**4**' and then toggled bold, it would un-un-bold the full selection.
Now it will bold the full selection (which is reflective of the way that highlighting '**1**23**4**' will not show the bold tool as active since the full selection is not bold

Reply:
Ah yes, let me look into what I can add in terms of testing